vuu-ui/packages/vuu-shell/src/density-provider/DensityProvider.tsx

```diff
@@ -0,0 +1,36 @@
+import { Dropdown, SelectionChangeHandler } from "@heswell/salt-lab";
```

Comment:
Why is the file DensityProvider.tsx when the exported component is DensitySwitch ?
File should be DensitySwitch.tsx